src/interactive.js

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable import/no-cycle */
+/* eslint-disable import/prefer-default-export */
+
```

Comment:
- [ ] Kindly remove all eslint disablement lines and try to fix eslint errors yourself.